static/app/utils/fields/index.ts

```diff
@@ -1362,6 +1364,11 @@ const REPLAY_FIELD_DEFINITIONS: Record<ReplayFieldKey, FieldDefinition> = {
     kind: FieldKind.FIELD,
     valueType: FieldValueType.STRING,
   },
+  [ReplayFieldKey.VIEWED_BY_ID]: {
+    desc: t('Sentry user id(s) who have seen this Replay'),
```

Comment:
this is a super nit but looks like replay isn't capitalized in the other descriptions 
```suggestion
    desc: t('Sentry user ID(s) who have seen this replay'),
```

Reply:
oh it is for URLS, but nowhere else

Reply:
oh i totally missed that! good catch, thanks for updating both